src/Sentry.Serilog/SentrySinkExtensions.cs

```diff
@@ -28,15 +31,269 @@ public static class SentrySinkExtensions
+            LogEventLevel? minimumEventLevel = null,
+            LogEventLevel? minimumBreadcrumbLevel = null,
+            IFormatProvider formatProvider = null)
+        {
+            return loggerConfiguration.Sentry(o => ConfigureSentrySerilogOptions(o,
+                sendDefaultPii,
+                isEnvironmentUser,
+                serverName,
+                attachStackTrace,
+                maxBreadcrumbs,
+                sampleRate,
+                release,
+                environment,
+                dsn,
+                maxQueueItems,
+                shutdownTimeout,
+                decompressionMethods,
+                requestBodyCompressionLevel,
+                requestBodyCompressionBuffered,
+                debug,
+                diagnosticsLevel,
+                reportAssemblies,
+                deduplicateMode,
+                initializeSdk,
+                minimumEventLevel,
+                minimumBreadcrumbLevel,
+                formatProvider));
+        }
+
+        public static void ConfigureSentrySerilogOptions(
+            SentrySerilogOptions sentrySerilogOptions,
+            bool? sendDefaultPii = null,
+            bool? isEnvironmentUser = null,
+            string serverName = null,
+            bool? attachStackTrace = null,
+            int? maxBreadcrumbs = null,
+            float? sampleRate = null,
+            string release = null,
+            string environment = null,
+            string dsn = null,
+            int? maxQueueItems = null,
+            TimeSpan? shutdownTimeout = null,
+            DecompressionMethods? decompressionMethods = null,
+            CompressionLevel? requestBodyCompressionLevel = null,
+            bool? requestBodyCompressionBuffered = null,
+            bool? debug = null,
+            SentryLevel? diagnosticsLevel = null,
+            bool? reportAssemblies = null,
+            DeduplicateMode? deduplicateMode = null,
+            bool? initializeSdk = null,
+            LogEventLevel? minimumEventLevel = null,
+            LogEventLevel? minimumBreadcrumbLevel = null,
+            IFormatProvider formatProvider = null)
+        {
+            if (sendDefaultPii.HasValue)
+            {
+                sentrySerilogOptions.SendDefaultPii = sendDefaultPii.Value;
+            }
+
+            if (isEnvironmentUser.HasValue)
+            {
+                sentrySerilogOptions.IsEnvironmentUser = isEnvironmentUser.Value;
+            }
+
+            if (!string.IsNullOrWhiteSpace(serverName))
+            {
+                sentrySerilogOptions.ServerName = serverName;
+            }
+
+            if (attachStackTrace.HasValue)
+            {
+                sentrySerilogOptions.AttachStacktrace = attachStackTrace.Value;
+            }
+
+            if (maxBreadcrumbs.HasValue)
+            {
+                sentrySerilogOptions.MaxBreadcrumbs = maxBreadcrumbs.Value;
+            }
+
+            if (sampleRate != null) // .HasValue instead?
```

Comment:
No strong opinion, [there's one occurance of `HasValue` in the code base](https://github.com/getsentry/sentry-dotnet/search?q=hasvalue&unscoped_q=hasvalue) so mostly `!= null` so far. Arguably `is null` and `HasValue` are "better" because `!=` is an operator.

But mostly since you used `HasValue` below, for consistency:

```suggestion
            if (sampleRate.HasValue)
```